extensions/artifacts/org.wso2.developerstudio.codenvy.ext.appfactory/src/main/java/org/wso2/developerstudio/codenvy/ext/appfactory/client/AppFactoryExtension.java

```diff
@@ -22,33 +22,39 @@
 import com.codenvy.ide.api.action.DefaultActionGroup;
 import com.google.inject.Inject;
 import com.google.inject.Singleton;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.wso2.developerstudio.codenvy.core.shared.CoreExtConstants;
 import org.wso2.developerstudio.codenvy.ext.appfactory.client.action.OpenPerspectiveAction;
-import org.wso2.developerstudio.codenvy.ext.appfactory.shared.AppFactoryExtConstants;
+import org.wso2.developerstudio.codenvy.ext.appfactory.shared.AppFactoryExtensionConstants;
 
 @Singleton
-@Extension(title = CoreExtConstants.EXT_NAME_PREFIX + AppFactoryExtConstants.EXT_NAME,
-           version = AppFactoryExtConstants.EXT_VERSION)
+@Extension(title = CoreExtConstants.EXT_NAME_PREFIX + AppFactoryExtensionConstants.EXT_NAME,
+        version = AppFactoryExtensionConstants.EXT_VERSION)
 public class AppFactoryExtension {
-
-	@Inject
-	public AppFactoryExtension(ActionManager actManager, OpenPerspectiveAction openAFAction,
-	                           NotificationManager notificationManager) {
-
-		try {
-			DefaultActionGroup wso2ActionGroup = (DefaultActionGroup) actManager
-					.getAction(CoreExtConstants.WSO2_ACTION_GROUP_ID);
-			actManager.registerAction(AppFactoryExtConstants.OPEN_AF_PERSPECTIVE_ACTION_ID,
-			                          openAFAction);
-			wso2ActionGroup.add(openAFAction);
-
-		} catch (Exception ex) {
-			notificationManager
-					.showNotification(new Notification("App Factory tools initiation failed.",
-					                                   Notification.Type.ERROR));
-		}
-
-	}
+    private static final Logger logger = LoggerFactory.getLogger(OpenPerspectiveAction.class);
+
+    @Inject
+    public AppFactoryExtension(ActionManager actManager, OpenPerspectiveAction openAFAction,
+                               NotificationManager notificationManager) {
+        try {
+            DefaultActionGroup wso2ActionGroup = (DefaultActionGroup) actManager
+                    .getAction(CoreExtConstants.WSO2_ACTION_GROUP_ID);
+            actManager.registerAction(AppFactoryExtensionConstants.OPEN_AF_PERSPECTIVE_ACTION_ID,
+                    openAFAction);
+            wso2ActionGroup.add(openAFAction);
+
+            if (logger.isDebugEnabled()) {
+                logger.debug("App Factory Extension successfully activated");
+            }
+        } catch (Exception e) {
```

Comment:
Hi Nuwan,
We fixed this issue by removing the try/ catch block since there were no exceptions throwing from the piece of code. You can look into the latest code at https://github.com/wso2/cloud-dev-studio/blob/master/extensions/artifacts/org.wso2.developerstudio.codenvy.ext.appfactory/src/main/java/org/wso2/developerstudio/codenvy/ext/appfactory/client/AppFactoryExtension.java